monkey/agent_plugins/build_plugin.sh

```diff
@@ -21,20 +21,25 @@ command -v pyenv >/dev/null || export PATH="$PYENV_ROOT/bin:$PATH"
 eval "$(pyenv init -)"
 
 pushd "$PLUGIN_PATH" || fail "$PLUGIN_PATH does not exist"
+VENDOR_DIR=vendor
+if [ -d "src/vendor-windows" ]; then
+    VENDOR_DIR=vendor-linux
+fi
+
 pip install pipenv
 pipenv requirements >> requirements.txt
-pip install -r requirements.txt -t src/vendor
+pip install -r requirements.txt -t src/$VENDOR_DIR
 rm requirements.txt
 
 # Package everything up
 pushd "$PLUGIN_PATH/src" || fail "$PLUGIN_PATH/src does not exist"
+SOURCE_ARCHIVE=$PLUGIN_PATH/$SOURCE_FILENAME
+tar -cf "$SOURCE_ARCHIVE" --exclude __pycache__ --exclude .mypy_cache --exclude .pytest_cache --exclude .git --exclude .gitignore --exclude .DS_Store -- *
```

Comment:
This commit mixes two things. One thing is building more complete source archives by using `--exclude` and `*`, another thing is installing requirements to the correct vendor directory.